### PR TITLE
Cleaner for unused old EBS Volumes

### PR DESCRIPTION
**Pull request by Paramadon**
# Description of the issue
There are thousands of ebs volumes not being used. This costs us about $10k a month in our test account. 

# Description of changes
This cleans up volumes that are at least a week old, status is "available", and they do not have any attachments to the volume (meaning the volume is unused). 
### There are over 1900 volumes before clean up
Screenshot 2024-02-29 at 4 28 25 PM

### Only 48 were actually in-use so the rest are deleted
Screenshot 2024-02-29 at 4 28 59 PM
# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
Passes actions and deletes only unused ebs volumes

### Passing tests
Screenshot 2024-02-29 at 4 30 29 PM

# Requirements
_Before commit the code, please do the following steps._
1. Run `make fmt` and `make fmt-sh`
2. Run `make lint`

### CODE REVIEW

File: tool/clean/clean_ebs/clean_ebs.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+// Clean eks clusters if they have been open longer than 7 day
+func main() {
+	err := cleanVolumes()
+	if err != nil {
+		log.Fatalf("errors cleaning %v", err)
+	}
+}
+
+func cleanVolumes() error {
+	log.Print("Begin to clean EKS Clusters")
```
Comment (sethAmazon) [inline tool/clean/clean_ebs/clean_ebs.go:26]: nit: EBS volumes

**Review by sethAmazon — APPROVED** [review]
lgtm

File: tool/clean/clean_ebs/clean_ebs.go (hunk 1 above)
Comment (Paramadon) [reply tool/clean/clean_ebs/clean_ebs.go]: changed

File: tool/clean/clean_ebs/clean_ebs.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+
+	"github.com/aws/amazon-cloudwatch-agent/tool/clean"
+)
+
+// Clean eks clusters if they have been open longer than 7 day
```
Comment (sethAmazon) [inline tool/clean/clean_ebs/clean_ebs.go:19]: not: EBS volumes

**Review by zhihonl — APPROVED** [review]

**Pull request merged**